elements/rocky-container-pulp/8.6-pulp-mofed

```diff
@@ -0,0 +1,130 @@
+# syntax=docker/dockerfile:1.4.3
+
+# To build this image you need enable docker build kit. This is for the heredoc support.
+
+FROM docker.io/library/rockylinux:8
+
+ARG rocky_version=8.6
+
+# /etc/machine-id needs to be populated for /bin/kernel-install to
+# correctly copy kernels into /boot.  We will clear this out in the
+# final image.
+RUN systemd-machine-id-setup
+
+RUN curl -L https://www.centos.org/keys/RPM-GPG-KEY-CentOS-SIG-Storage -o /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Storage\
+  && curl -L https://www.mellanox.com/downloads/ofed/RPM-GPG-KEY-Mellanox -o /etc/pki/rpm-gpg/RPM-GPG-KEY-Mellanox\
+  && curl -L https://download.docker.com/linux/centos/gpg -o /etc/pki/rpm-gpg/RPM-GPG-KEY-docker
+
+COPY <<Rocky-BaseOS.repo <<Rocky-Extras.repo <<Rocky-AppStream.repo <<Rocky-PowerTools.repo <<Rocky-NFV.repo <<ceph.repo <<docker.repo /etc/yum.repos.d/
+[baseos]
+name=Rocky Linux $releasever - BaseOS
+enabled=1
+baseurl=https://pulp.128-232-222-245.sslip.io/pulp/content/rocky/8.6/BaseOS/x86_64/os/20220914T080246/
```

Comment:
This really should somehow be a variable

Reply:
Seems like the best way would be if we had build arg support in DIB for the containerfile element?